classes/Assets.php

```diff
@@ -584,6 +584,22 @@ public function add_identifier_class_to_body( $classes ) {
 			$to_add[] = 'tutor-screen-frontend-dashboard';
 		}
 
+		if ( is_post_type_archive( 'courses' ) ) {
+			$to_add[] = 'tutor-frontend';
+		}
+
+		if ( tutor_utils()->is_tutor_frontend_dashboard() ) {
+			$to_add[] = 'tutor-frontend';
+		}
+
+		if ( is_single() ) {
+			global $post;
+			$post_types = array( 'lesson', 'courses', 'tutor_quiz', 'tutor_assignments', 'tutor-google-meet', 'tutor_zoom_meeting' );
```

Comment:
Please use the `tutor()` function to get post types instead of writing explicitly